FirebaseStorageSwift/Sources/SwiftAPIExtension.swift

```diff
@@ -14,8 +14,14 @@
 
 import FirebaseStorage
 
-/// getResultCallback generates a closure that returns a Result type from a closure that returns an
-/// optional type and Error.
+/// Generates a closure that returns a Result type from a closure that returns an optional type and
+/// Error.
+///
+/// - Parameters:
+///   - completion: A completion block returning a Result enum with either a generic object or
+///                 an NSError.
```

Comment:
Code references (like `NSError`) really should be in backticks, but I can see that this file is no worse off than the original storage headers.

In Swift shouldn't we substitute `Error` for `NSError`?